SwiftParcel.Services.Parcels/src/SwiftParcel.Services.Parcels.Infrastructure/SwiftParcel.Services.Parcels.Infrastructure/Mongo/Documents/Extensions.cs

```diff
@@ -124,7 +126,8 @@ public static ParcelDto AsDto(this ParcelDocument document)
                 VipPackage = document.VipPackage,
                 CreatedAt = document.CreatedAt,
                 ValidTo = document.ValidTo,
-                CalculatedPrice = document.CalculatedPrice
+                CalculatedPrice = document.CalculatedPrice,
+                PriceBreakDown = document.PriceBreakDown.AsDto()
```

Comment:
In my case I have to change this line to: "PriceBreakDown = document.PriceBreakDown?.AsDto()" because during getting parcels I could observe error presented below.
![problem1](https://github.com/SaintAngeLs/courier_app/assets/148250180/bc28c14d-5311-45e3-9da4-be6ff8a72379)

Reply:
It's probably because in our database there are parcels with old structure, without pricebreakdown. The same thing goes for orders.
After deleting them everything should be fine.